### ISSUE TITLE
inverted/exclusive portals

### ISSUE THREAD

**Issue by Gravemind2401** [Multiverse/Multiverse-Portals#113]
currently (to my knowledge) all portals created with MV2 are inclusive - they only teleport players that walk into them. it would be useful in some situations to have an exclusive portal type, that only teleports players that leave the portal area (ie: to keep players in a certain zone, and if they try to leave it would teleport them back in, or to somewhere else).
would this be able to be added as a feature?

**Comment by benwoo1110** [comment]
I dont think this make much sense to be a portal. This sounds more like a worldguard region, where you can configure it to run a command when the players leaves the region.
